cliquet/authorization.py

```diff
@@ -49,6 +61,15 @@ class RouteFactory(object):
     }
 
     def __init__(self, request):
+        # Prefix the user id with the authn policy type name.
+        user_id = request.authenticated_userid
+        # This comes from ``cliquet.initialization.setup_authentication()``.
+        authn_type = getattr(request, 'authn_type', None)
+        self.prefixed_userid = None
+        if user_id and authn_type:
+            self.prefixed_userid = '%s:%s' % (authn_type.lower(), user_id)
```

Comment:
Nevermind, I just released cliquet-fxa 1.1.0 and we'll consider that cliquet-fxa was never used by anybody (only mozilla apps can use it)